src/pages/Game/Game.tsx

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-self-compare */
+import React, { useState, useEffect, useMemo } from "react";
+import { ToastContainer, toast } from "react-toastify";
+import { useSelector, useDispatch } from "react-redux";
+import { useNavigate } from "react-router-dom";
+//images
+import bomber from "images/img-game/13026.png";
+import smileNotRight from "images/img-game/smile-sad.png";
+//Components
+import Button from "сomponents/Button";
+//redux state
+import { RootState } from "redux/store";
+//reducer
+import { makeMove } from "redux/store";
+//helper
+import { mockedFunction } from "helpers";
+//style
+import style from "./style.module.scss";
+import "react-toastify/dist/ReactToastify.css";
+
+const Game = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const [isNotMines, setIsNotMines] = useState<boolean>(true);
```

Comment:
it is not a  good decision to use `isNotMines` and `isMines` simultaneously . Use only one of that